packages/axios-extension/src/abap/adt-catalog/services/ui5-rt-version-service.ts

```diff
@@ -0,0 +1,41 @@
+import type { AdtCategory } from '../../types';
+import { AdtService } from './adt-service';
+
+/**
+ * UI5RtVersionService implements ADT requests to get UI5 Version
+ * of ABAP system.
+ */
+export class UI5RtVersionService extends AdtService {
+    /**
+     * @see AdtService.getAdtCatagory()
+     */
+    private static adtCategory = {
+        scheme: 'http://www.sap.com/adt/categories/filestore',
```

Comment:
Updating to `https://www.sap.com/adt/categories/filestore` should resolve sonar issue

Reply:
All services schema is "http". Also update schema to "https" cause test failure. Is there some workaround how I can disable sonar specially for this piece of code?